assets/i18n/es.json

```diff
@@ -175,6 +175,18 @@
   "I18N_PLAYER_RATE_EXPLORATION": "¿Has aprendido algo nuevo? ¿Cómo calificarías esta exploración?",
   "I18N_PLAYER_RATINGS_TOOLTIP": "Calificaciones",
   "I18N_PLAYER_RECOMMEND_EXPLORATIONS": "Otras exploraciones que tal vez te gusten",
+  "I18N_PLAYER_REPORT_MODAL_BODY_AD": "Engañosa contenido, spam o anuncios",
```

Comment:
@mit0110 @joshuacano @anafrancisca Please could you take a look at these translations, and let us know if any changes are needed? They are for new functionality which allows users to flag problematic explorations for moderator attention.

Thanks!